DancingGoat/Areas/Admin/AppSettingProvider.cs

```diff
@@ -178,5 +181,40 @@ public static string KenticoCloudUrl
                 }
             }
         }
+
+        public static string[] ResponsiveWidths
+        {
+            get
```

Comment:
It would be useful to simplify the getter:
```
if (_responsiveWidths == null)
{
    var setting = ConfigurationManager.AppSettings[RESPONSIVE_WIDTHS];
    _responsiveWidths = string.IsNullOrEmpty(setting) ? new string[] { } : setting.Split(',');
}
return _responsiveWidths;
```
Or to use a Lazy for the field:
```
private static Lazy<IEnumerable<string>> _responsiveWidths = new Lazy<IEnumerable[]>(() => 
{
    var setting = ConfigurationManager.AppSettings[RESPONSIVE_WIDTHS];
    return setting?.Split(',') ?? new string[0];
});
```
and simplify the getter that way
```
public static IEnumerable<string> ResponsiveWidths => _responsiveWidths.Value;
```